spatialmath/box.go

```diff
@@ -118,7 +126,6 @@ func (b *box) CollidesWith(g Geometry) (bool, error) {
 	return true, newCollisionTypeUnsupportedError(b, g)
 }
 
-// CollidesWith checks if the given box collides with the given geometry and returns true if it does.
```

Comment:
nit: This comment is a duplicate of the comment above and isn't actually for the `DistanceFrom` function; it's for the `CollidesWith` function.